examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/CastingServerBridge.h

```diff
@@ -109,24 +110,20 @@
 
  @param clientQueue Queue to dispatch the call to the commissioningWindowRequestedHandler on
 
- @param commissioningCompleteCallback Callback for when commissioning of this app has been completed  via a call to the general
- commissioning cluster (by usually an on-network TV/Media device acting as a Matter commissioner)
-
  @param onConnectionSuccessCallback Handles a VideoPlayer * once connection is successfully established
 
  @param onConnectionFailureCallback Handles MatterError if there is a failure in establishing connection
 
  @param onNewOrUpdatedEndpointCallback Handles a ContentApp * for each new ContentApp is found. May be called multiple times based
  on the number of ContentApp
 
- @param commissioningWindowRequestedHandler Handler to call on requesting the opening of a commissioning window
+ @param commissioningCallbackHandlers Optional parameter to specific handlers for callbacks during commissioning
```

Comment:
nit, can you order these in the same order that the function has them?

Reply:
Done